string_scrap.js

```diff
@@ -1,31 +1,43 @@
+//1
 let word = 'PURSUIT';
 console.log(word[0]);
 
+//2
 let str = 'PURSUIT';
 console.log(str.length);
 
+//3
 let name = 'PURSUIT';
 let lastCHar = name.charAt(name.length - 1);
 console.log(lastCHar);
 
+//4
```

Comment:
Question was asking for this output => "jimmY". This code would only give us "y". Look into slice method to solve this problem.